README.md

```diff
@@ -47,24 +47,21 @@ You need to install the following:
 * [Python](www.python.org) - the platform needs at least Python 3.7.0
 * [Pipenv](https://pypi.org/project/pipenv/)
 * [Docker](https://docs.docker.com/install/)
-* Amazon Web Services account and [AWS command line tools]()
+* Amazon Web Services account and [AWS command line tools](https://aws.amazon.com/cli/)
+* [Git Bash](https://gitforwindows.org/) - you'll need this installed to run various shell scripts in the project if you are running on Windows. Also ensure that `.sh` is set to be associated with Git Bash or script execution will fail.
 
 ## Terraform workspaces and unique names
 
-22 character
 
 It is advised that you use a separate [terraform workspace](https://www.terraform.io/docs/enterprise/workspaces/index.html) across each part of the project, if you are collaborating with others on the same AWS account.
 
-The workspace name should be unique to you, within the group of users you are collaborating with on this project.  
-
-
-is used in a variety of places as identifiers - to cater for limits of the AWS components that are used, you should use a name that is **16 characters or less**.
+The workspace name should be unique to you, within the group of users you are collaborating with on this project.  There is an AWS size limitation, where the total length of the workspace name and app name need to be **22 characters or less**.
```

Comment:
Is that right? If we use longer names then this would go down?

Reply:
yeah the limitation is the ES domain (28 characters max), where we do d-<workspace_name>-<app_name>-es

Reply:
I've added that as a note in the docs so there's context